pkg/client/kubernetes/client.go

```diff
@@ -57,6 +76,7 @@ func NewClientFromBytes(kubeconfig []byte) (Client, error) {
 	if err != nil {
 		return nil, err
 	}
+	config.Transport = newTransport()
```

Comment:
Just saw it: Instead of assigning `config.Transport` here, we should assign it in `newClientSet` - this is our most 'low-level' method before creating a `kubernetes.Interface`

Reply:
so you mean something like that 
```
config.Transport = newTransport()
clientset, err := kubernetes.NewForConfig(config)
```
in the `newClientSet` method?